.github/workflows/codeRiewson.yml

```
@@ -14,33 +14,52 @@ jobs:
+              messages: [
+                {role: "user", content: $additionalMessage},
+                {role: "user", content: $content}
+              ]
+            }')
+  
+            # Send request to OpenAI API
+            response=$(curl -s https://api.openai.com/v1/chat/completions \
+              -H "Content-Type: application/json" \
+              -H "Authorization: Bearer ${{ secrets.OPENAI_API_KEY }}" \
+              -d "$json_payload")
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)